app/ui/src/assets/dictionary/en-GB.json

```diff
@@ -38,7 +38,15 @@
         "create-review-step-title": "Review Actions",
         "create-security-step-title": "Specify Security",
         "create-details-step-title": "Review/Edit Connector Details",
-        "edit-tooltip": "{{project.name}} cannot make this specification available for review/edit. If you want to review or edit the API before {{project.name}} creates the connector, try manually downloading the specification and then upload that file."
+        "edit-tooltip": "{{project.name}} cannot make this specification available for review/edit. If you want to review or edit the API before {{project.name}} creates the connector, try manually downloading the specification and then upload that file.",
+        "api-file-upload": "Upload an OpenAPI file",
+        "api-file-upload-dnd": "Drag and drop your OpenAPI file here or click the 'Choose File' button to use the file chooser",
+        "api-upload-choose-method": "Choose how you want to create your connector:",
+        "url-upload-instructions": "Upload the OpenAPI 2.0 file for your custom API client connector. Custom APIs are RESTful APIs and can be hosted anywhere, as long as a well-documented API specification is available and conforms to the OpenAPI standard.",
+        "api-upload-invalid-file": "'{{0}}' is not a valid file. Only files that end in '.json' can be imported.",
+        "api-url-upload": "Use a URL",
+        "api-url-upload-note": "* Note: we will download this for you but any updates to the online version will not be updated in our app.",
+        "url-validation-error": "You must enter a valid URL before submitting the form"
```

Comment:
There should be a period at the end of that message since it is a complete sentence.